confidential-data-hub/golang/pkg/api/cdhgrpc/cdhgrpc.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
```

Comment:
Can we reuse the proto already defined in https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/hub/protos/api.proto? This could help with so-called "single truth of source".

Reply:
Done.